04.bowling/bowling.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+shots = []
+ARGV[0].split(',').each do |score|
+  if score == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << score.to_i
+  end
+end
```

Comment:
ちょっと上級テクニックですが、こんな風に書くこともできますー。

```suggestion
shots = ARGV[0].split(',').flat_map do |score|
  score == 'X' ? [10, 0] : score.to_i
end
```